pipeline/00-ingest.R

```diff
@@ -178,7 +177,7 @@ hie_data_training_sparse <- hie_data %>%
     replacement_source = any_of(chars_cols$replace$source)
   ) %>%
   mutate(
-    ind_pin_is_multicard = "0",
+    ind_pin_is_multicard = FALSE,
```

Comment:
Setting this to `0` was causing a join error. Changing it to a boolean resolved this.